config/locales/en.yml

```diff
@@ -3586,8 +3582,10 @@ See the %{link} to find out more about %{sitename}'s features and to start using
           or_enter_new_card: "Or, enter details for a new card:"
           remember_this_card: Remember this card?
     date_picker:
-      format: ! '%Y-%m-%d'
-      js_format: 'yy-mm-dd'
+      flatpickr_date_format: "Y-m-d"
+      flatpickr_datetime_format: "Y-m-d H:i"
+      today: "Today"
+      now: "Now"
```

Comment:
We'll need to make sure all instances are aware of this change and it gets translated before deploying.

Reply:
Yes. How can we be sure of that? Is this something I need to check in the future? What should I do?

Reply:
we should let the next release manager know so we wait for translators and include their translations in the release.

Reply:
The next release manager is @Matt-Yorkley. So, I let you know Matt!